### ISSUE TITLE
FR "Turn off metronome after X bars"

### ISSUE THREAD

**Issue by Jeff0S** [reaper-oss/sws#30]
_From [swstim](https://code.google.com/u/swstim/) on November 16, 2009 03:47:57_

From billoon @ forum:

Is it possible to make a 'run metronome for [x] bars setting', that would
turn off the metronome after playback/recording gets to [x] bars?

_Original issue: http://code.google.com/p/sws-extension/issues/detail?id=30_

**Comment by X-Raym** [comment]
@swstim indeed, marker actions seems way more flexible. I think it can be closed.
